linux/src/async_socket_linux.c

```diff
@@ -370,12 +380,14 @@ static int initialize_global_thread(void)
     // Codes_SRS_ASYNC_SOCKET_LINUX_11_009: [ If the g_thread_access_cnt count is 1, initialize_global_thread shall do the following: ]
     if (current_count == 1)
     {
+        interlocked_increment(&g_epoll_access);
+
```

Comment:
I think generally this reference counting is not needed. What is the case that fails if we don't have it?